### ISSUE TITLE
beeb  me for waiting game

### ISSUE THREAD

**Issue by brw31415** [online-go/online-go.com#631]
### Summary
many times you open a game, you go somewhere else to wait and when ,somebody joins and you dont notice. i would love to have it beeb me to join the game i started
peace 

*Briefly describe the issue you are reporting*

### Environment

 * Operating system: 
 * Browser:
 * Browser Version:

### Steps To Reproduce:

1. Go to ...
2. Click ...
3. See that X wrong 

### Additional Information:

*Include any screen shots, links to games/reviews, wonky SGFs, or other information that is useful*

**Comment by GreenAsJade** [comment]
Implemented by #650 

Note: I'm not entirely sure why this is marked as a "bug".  It's a feature that is frequently requested in the forums - I've always assumed that it just wasn't implemented yet.   

I couldn't find anywhere where it "tries to implement and is buggy".

So I did a from scratch implementation.  It feels right to me anyhow: I'm not sure this would want to be tangled up in Goban.   It's about getting to the Goban, not the Goban itself :)
